example-get-started/code/README.md

```diff
@@ -33,10 +33,17 @@ $ source .env/bin/activate
 $ pip install -r src/requirements.txt
 ```
 
+Note that dvc is not installed by the requirements file, 
```

Comment:
let's make it a 

> Note ....

and move to the very top of the section

also, text should be something along those lines:

"This instruction assumes that DVC is already installed, see the installation guide and pick the best way to install it that works for you"